src/ops/TriggerOps.ts

```diff
@@ -88,9 +88,11 @@ export default class TriggerOps extends BaseOp {
             command.metadata = commandRequest.metadata;
 
             this.postTrigger(requestBody).then((res:Response)=>{
-                var trigger = new Trigger(requestObject.predicate, command, null);
-                trigger.triggerID = (<any>res).body.triggerID;
-                trigger.disabled = false;
+                var trigger = new Trigger(
+                    (<any>res).body.triggerID,
+                    requestObject.predicate,
```

Comment:
In thing-if AndroidSDK and thing-if iOSSDK, `postTrigger()` method gets trigger from server with `getTrigger()` method. Why do not we get trigger with `getTrigger` in thing-if JSSDK?